github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -0,0 +1,52 @@
+periodics:
+- name: periodic-kubevirtci-sync-crio-repository-weekly
+  interval: 168h
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  spec:
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        env:
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/gcs/service-account.json
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - >
+            set -e;
+            BUCKET=kubevirtci-crio-mirror
+            LOCAL_MIRROR_DIR=opensuse-mirror
+            OS=CentOS_8_Stream
+            CRIO_VERSION=1.20
+
+            DEVEL_LIBCONTAINERS_STABLE=devel_kubic_libcontainers_stable
+            curl -L -o $DEVEL_LIBCONTAINERS_STABLE.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/devel:kubic:libcontainers:stable.repo
+            reposync -c $DEVEL_LIBCONTAINERS_STABLE.repo -p ./$LOCAL_MIRROR_DIR -n --repoid=$DEVEL_LIBCONTAINERS_STABLE
+
+            sync_crio_repo_for_version () {
+              DEVEL_LIBCONTAINERS_STABLE_CRIO=devel_kubic_libcontainers_stable_cri-o_$1
+              curl -L -o DEVEL_LIBCONTAINERS_STABLE_CRIO.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$1/$OS/devel:kubic:libcontainers:stable:cri-o:$1.repo
+              reposync -c DEVEL_LIBCONTAINERS_STABLE_CRIO.repo -p ./$LOCAL_MIRROR_DIR -n --repoid=DEVEL_LIBCONTAINERS_STABLE_CRIO
+            }
+            sync_crio_repo_for_version $CRIO_VERSION
+
+            gsutil mb gs://$BUCKET || true
```

Comment:
I would remove this, we need to configure the bucket anyway to allow serve files with http from it.

Reply:
Removed